bindings/python/cntk/ops/__init__.py

```diff
@@ -263,18 +263,84 @@ def labels_to_graph(labels, name=''):
 @typemap
 def forward_backward(graph, features, blankTokenId, delayConstraint=-1, name=''):
     '''
-    Criterion node for training methods that rely on forward-backward Viterbi-like passes, e.g. Connectionist Temporal Classification (CTC) training
-    The node takes as the input the graph of labels, produced by the labels_to_graph operation that determines the exact forward/backward procedure.
+    Criterion node for training methods that rely on forward-backward Viterbi-like passes,
+    e.g. Connectionist Temporal Classification (CTC) training
+    The node takes as the input the graph of labels, produced by the labels_to_graph
+    operation that determines the exact forward/backward procedure.
+
+    Example:
+            graph = cntk.labels_to_graph(labels)
+            networkOut = model(features)
+            fb = C.forward_backward(graph, networkOut, 132)
+
+    This op requires that both graph and features have the same dynamic sequence axis (i.e. same sequence length).
+
+    Labels feed into cntk.labels_to_graph and hence to forward_backward are represented as 1-hot vectors
+    for each frame. The 1-hot vectors may have either value 1 or 2 at the position of the phone
+    corresponding to the frame, where the value 1 means the frame is within phone boundary (i.e. not the
+    first frame of the phone) and 2 means the frame is the phone boundary (i.e. is the first frame of the phone).
+    If you are using HTKFeatureDeserializer, this encoding will be automatically done.
```

Comment:
"HTKFeatureDeserializer" => "HTKMLFDeserializer"

Reply:
Done, sorry for the lapse!